jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/support/AbstractPortsExtractorTest.java

```diff
@@ -100,10 +111,9 @@ public void testNoFile() {
 
     @Test
     public void testConfigFileDoesNotExist() {
+        logger = mock(PrefixedLogger.class);
         final String nonExistingFile = "/bla/blub/lalala/config.yml";
-        new Expectations() {{
-            logger.warn(anyString, withEqual(FileUtil.getAbsolutePath(nonExistingFile)));
-        }};
+        doNothing().when(logger).warn(anyString(), eq(FileUtil.getAbsolutePath(nonExistingFile)));
```

Comment:
should not be necessary if using a real PrefixedLogger instance as suggested in previous comments